composer.json

```diff
@@ -21,7 +21,7 @@
     }
   },
   "require": {
-    "php": ">=8.1",
+    "php": ">=8.2",
```

Comment:
Just flagging that UK Loans using this lib and their PHP is 8.1 - they will not be able to use the new version of this lib before they upgrade their PHP

Reply:
aye, but we have https://lendable-uk.atlassian.net/browse/LOAN-13041 on the board - hopefully it will not take long for us to get there :smiley:

Reply:
Sounds good to me, just wanted to make sure that we took it into account, thanks!